urdf_parser_plugin/include/urdf_parser_plugin/parser.h

```diff
@@ -37,13 +37,13 @@
 #ifndef URDF_PARSER_PLUGIN_H
 #define URDF_PARSER_PLUGIN_H
 
-#include <urdf/urdfdom_compatibility.h>
+#include <urdf_world/types.h>
 
```

Comment:
Nit: `#include <string>` here.

Reply:
Included `<string>` in 7efedb4a0426901779465b37f794fbf138d31947